react/Image.tsx

```diff
@@ -81,15 +79,20 @@ function Image(props: ImageProps) {
     promotionId,
     promotionName,
     promotionPosition,
+    classes,
   } = props
+
   const imageRef = useRef<HTMLImageElement | null>(null)
   const isLoaded = useImageLoad(imageRef, {
     bailOut: !experimentalPreventLayoutShift,
   })
+
   const intl = useIntl()
-  const handles = useCssHandles(CSS_HANDLES, {
+  const { handles } = useCssHandles(CSS_HANDLES, {
     migrationFrom: 'vtex.store-components@3.x',
```

Comment:
Is this working properly?

Reply:
Good question, let me check

Reply:
There's an image [here](https://icaroshopbycategory--storecomponents.myvtex.com/about-us) (VTEX logo). I've added custom classes to the image component and the behavior is that it overrides both the default handle and the migrationFrom handle.

<img width="340" alt="Screen Shot 2020-12-15 at 6 57 44 PM" src="https://user-images.githubusercontent.com/8127610/102277701-6c589e00-3f07-11eb-8ffc-4c26a5b5ebee.png">

<img width="246" alt="Screen Shot 2020-12-15 at 6 59 37 PM" src="https://user-images.githubusercontent.com/8127610/102277920-b477c080-3f07-11eb-87e4-1559c21c204b.png">